lib/matplotlib/backends/backend_wx.py

```diff
@@ -1345,15 +1345,15 @@ def __init__(self, parent, help_entries):
                 grid_sizer.Add(label, 0, 0, 0)
         # finalize layout, create button
         sizer.Add(grid_sizer, 0, wx.ALL, 6)
-        OK = wx.Button(self, wx.ID_OK)
-        sizer.Add(OK, 0, wx.ALIGN_CENTER_HORIZONTAL | wx.ALL, 8)
+        ok = wx.Button(self, wx.ID_OK)
+        sizer.Add(ok, 0, wx.ALIGN_CENTER_HORIZONTAL | wx.ALL, 8)
         self.SetSizer(sizer)
         sizer.Fit(self)
         self.Layout()
-        self.Bind(wx.EVT_CLOSE, self.OnClose)
-        OK.Bind(wx.EVT_BUTTON, self.OnClose)
+        self.Bind(wx.EVT_CLOSE, self._on_close)
+        ok.Bind(wx.EVT_BUTTON, self._on_close)
 
-    def OnClose(self, event):
```

Comment:
Wait, this one isn't a binding, and didn't match the old name.

Reply:
It's just a normal method on a private class, so I renamed it at the same time.

Reply:
Yep, oops, I missed the binding on this one.